sysdiff detects gce-workload-cert-refresh.timer

**What happened:**

In today's nightly this popped up:

```
_______________________________________________ test_sysdiff_after_tests _______________________________________________
integration/core/test_sysdiff.py:59: in test_sysdiff_after_tests
    pytest.fail(
E   Failed: System changes were detected during the test run. See stderr output for details.
------------------------------------------------- Captured stderr call -------------------------------------------------
System changes detected - detailed diff:
=== Systemd unit state changes (2026-04-02T10:36:43.769227-root-before-tests -> 2026-04-02T10:43:16.005669-root-after-tests) ===
--- systemd_units@snapshot_a
+++ systemd_units@snapshot_b
@@ -35,6 +35,7 @@
 auditd.service	loaded	active	running
 dbus.service	loaded	active	running
 dracut-shutdown.service	loaded	active	exited
+gce-workload-cert-refresh.service	loaded	activating	start
 getty@tty1.service	loaded	active	running
 google-shutdown-scripts.service	loaded	active	exited
 ipvsadm.service	loaded	active	exited
@@ -155,7 +156,7 @@
 dpkg-db-backup.timer	loaded	active	waiting
 e2scrub_all.timer	loaded	active	waiting
 fstrim.timer	loaded	active	waiting
-gce-workload-cert-refresh.timer	loaded	active	waiting
+gce-workload-cert-refresh.timer	loaded	active	running
 google-oslogin-cache.timer	loaded	active	waiting
 logrotate.timer	loaded	active	waiting
 systemd-cron-cleaner.timer	loaded	active	waiting

Tip: lines starting with '+' mean new/changed; '-' mean removed/previous state.
```

This runs periodically and should be safe to ignore.

--- a/tests/plugins/sysdiff.py
+++ b/tests/plugins/sysdiff.py
@@ -69,6 +69,9 @@ IGNORED_SYSTEMD_PATTERNS = [
     "user-runtime-dir@*.service",
     "user@*.service",
     "user-*.slice",
+    # runs periodically
+    "gce-workload-cert-refresh.service",
+    "gce-workload-cert-refresh.timer",
 ]
 
 # Units to wait for before taking a snapshot (unit_name: expected_sub_state)

